app/assets/stylesheets/components/_input.scss

```diff
@@ -26,6 +26,10 @@ input[type="checkbox"] {
   }
 }
 
+.display-inline-block {
+  display: inline-block;
```

Comment:
Nice fix!! Tachyons has this property available as `dib`, so you can use that instead :)

Linking their table of properties bc it's helpful to cmd-F stuff, most common things exist arleady~
http://tachyons.io/docs/table-of-properties/